data-structures/ds-que/python3/queue.py

```diff
@@ -1,63 +1,56 @@
 class Queue():
     def __init__(self):
-        '''Initialize this queue to the empty queue.'''
+        """Initialize this queue to the empty queue."""
         self.queue = list()
         self.rear = -1
         self.front = 0
         self.size = 0
 
-    '''add element to the queue'''
-
     def enqueue(self, data):
-        '''If queue already full can't add element further'''
-        if(self.isFull()):
+        """If queue already full can't add element further."""
+        if self.is_full():
             print("Queue is full")
         else:
             self.rear += 1
             self.queue.insert(self.rear, data)
             self.size += 1
 
-    def isFull(self):
-        if(self.rear == 100):
+    def is_full(self):
+        if self.rear == 100:
             return 1
         else:
             return 0
 
-    def isEmpty(self):
-        if(self.size == 0):
+    def is_empty(self):
+        if self.size == 0:
             return 1
         else:
             return 0
 
-    '''remove front element in the queue'''
-
     def dequeue(self):
-
-        if(not self.isEmpty()):
+        if not self.is_empty():
             self.data = self.queue[self.front]
             self.front += 1
             self.size -= 1
             return self.data
         else:
             return -1
 
-    '''return front element no delete here'''
-
     def peek(self):
-        if(not self.isEmpty()):
+        """return front element no delete here."""
```

Comment:
Codacy found an issue: [First word of the first line should be properly capitalized ('Return', not 'return') (D403)](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6264809)